src/collector.ts

```diff
@@ -391,7 +384,18 @@ const addSigListeners = (): void => {
     console.log('Config reloaded', CONFIG)
   })
   process.on('SIGINT', async () => {
+    stopCheckpointVerificationProcess = true
     console.log('DETECTED SIGINT SIGNAL')
+    await stopRMQEventsConsumers()
+    await sleep(5000)
```

Comment:
waiting for 5s to allow processing in-flight messages or currently being processed if any